src/components/Footer.js

```diff
@@ -230,12 +128,12 @@ export class Footer extends Component {
 
   render(){
     const {members} = this.state;
-    let together = members.join(" | ");
+    let together = members.join(" ");
     
     return (
       <div>
-        <a href="https://github.com/gwg-women/gwg-women-techmakers"> Grow with Google Project</a>: <div dangerouslySetInnerHTML={{ __html: together}} />
-      </div>
+        <h1 className="gwg-link"><a href="https://github.com/gwg-women/gwg-women-techmakers"> Grow with Google Project</a></h1> <ul dangerouslySetInnerHTML={{ __html: together}} />
```

Comment:
If this is going to be a header, it should be an h2. H1 reserved for title of page/application.